### ISSUE TITLE
How about painting the form?

### ISSUE THREAD

**Issue by ericsia** [Onelio/Win32GUI#2]
same as the title. Maybe you can create another example to paint the windows form?
That would make more sense as gui windows form may contain color

**Comment by moyamejiasr** [comment]
Hi, you mean the window background colour right?
That's a good idea and seems easy enough to implement: https://docs.microsoft.com/en-us/windows/win32/gdi/drawing-a-custom-window-background
I'll give it a try whenever I have time to. You can however try implementing it by yourself if you want.

EDIT
______
Just noticed that I already implemented this. I just forgot to make wndclass brush NULL. Will upload a fix later.

**Comment by ericsia** [comment]
> Hi, you mean the window background colour right?
> That's a good idea and seems easy enough to implement: https://docs.microsoft.com/en-us/windows/win32/gdi/drawing-a-custom-window-background
> I'll give it a try whenever I have time to. You can however try implementing it by yourself if you want.
> 
> EDIT
> 
> Just noticed that I already implemented this. I just forgot to make wndclass brush NULL. Will upload a fix later.

Yes you are right. How about this? i notice your repo not yet provide a sample project.
Try clone this gui if you are able to http://medlexo.ml/ this design is not bad imo.
It will attract more people to your repo if you provided a working sample implementation.
